lib/rules/require-space-before-keywords.js

```diff
@@ -55,7 +55,7 @@ module.exports.prototype = {
             excludedKeywords = keywords.allExcept;
         }
 
-        keywords = defaultKeywords.filter(function(keyword) {
+        keywords = (keywords === true ? defaultKeywords : keywords).filter(function(keyword) {
```

Comment:
this will work but @hzoo pointed out that instead, this statement should instead be moved into the preceding "if" block.